.github/workflows/_ci.yaml

```diff
@@ -15,18 +15,43 @@ on:
       TAG_BASE:
         description: "Tags of the base image built"
         value: ${{ jobs.build-base.outputs.DOCKER_TAG }}
-      TAG_JAX:
-        description: "Tags of the JAX image built"
+      TAG_JAX_MEALKIT:
```

Comment:
Maybe we should just use a single JSON object to store all the mealkit/final tags, respectively. This is just a thought and please feel free to throw in yours :)

Reply:
Never used it before, but let me give a shot

Reply:
Not sure it looks better now, but more flexible I think